packages/localization/src/languages/en.json

```diff
@@ -153,7 +153,9 @@
             "feedback": {
                 "noChangesMade": "No changes have been made.",
                 "unknownError": "Unknown error",
-                "createdByNoun": "Created by {fullName} from the Noun project"
+                "createdByNoun": "Created by {fullName} from the Noun project",
```

Comment:
What is that ? Noun project?

Reply:
They created free SVG icon that we use, for example for device type display in device details, but we have to reference them.